### ISSUE TITLE
Sample Programs

### ISSUE THREAD

**Issue by ginonott** [FRUK-Simulator/Simulator#56]
To make development quicker and testing easier it would be nice to have selectable demo programs. For instance, a simple one that makes the robot go back and forth, in a circle, navigate based on sensor, etc.

Dependent on #32 most likely

**Comment by eurocoinsalbum** [comment]
I will try to have an initial demo on startup

**Comment by eurocoinsalbum** [comment]
The program in the PR is not meant to be ready ;-) and could improved stepwise.
It's more about if we can life for now with that kind of implementation before we have nice menus or resource loading via URL etc.
